### ISSUE TITLE
[BUG] !raidready queue isn't being flushed

### ISSUE THREAD

**Issue by shipofgenius** [TalimaVale/BeastieBot#21]
Seems to ignore people who've successfully attended a previous raid on the next !raidready

**Comment by scryptonite** [comment]
🤔 This was (or I am pretty sure it should've been) fixed in PR #20. Can you pull from the master here and see if it's still a problem?

**Comment by scryptonite** [comment]
@shipofgenius Oh, are you saying that you think the raid team should not include a username after witnessing that username raid the target channel? Right now teamTALIMA's master branch (and mine as well) should prevent you from double joining the raid team & also from re-joining the raid team while in the middle of the raid (during the raid prep and also the actual raid itself).

The raid team should then be cleared immediately after handing out the raid bonus so that the broadcaster could start another raid if they wanted to. Do you not think this is the correct behavior?

**Comment by shipofgenius** [comment]
Mmm Let me check again. I did test that raid was off by default. Apologies, didn't see that you addressed this to. Will test today.

Basically its the bug where if we've already been on a full successful raid, beastie doesnt flush us from the queue. So when we type !raidready for the next raid, nothing happens.

**Comment by shipofgenius** [comment]
UPDATE: Scryptonite fixed in a32821277ffb268614335548f445f7b53a0efc0b

New bug. Easy fix. debug() should be warn() or error():
```
[13:30] info: [#trugamer_maniac] <chia_bot>: !raidteam
[13:30] info: [#trugamer_maniac] <matrix_agent>: 1 teammates are prepared to raid! Use !raidready to join the raid team and receive bonus raid awesomeness!
[13:50] info: [#trugamer_maniac] Now hosting dugstanat for 0 viewer(s).
[13:50] info: Executing command: JOIN #dugstanat
[13:50] info: [#trugamer_maniac] <matrix_agent>: Time to raid! :D rawr
[13:50] info: Joined #dugstanat [13:50] info: Joined #dugstanat
[13:51] info: [#dugstanat] <trugamer_maniac>: Hi hi!

javascript\teamTalima\BeastieBot\commands\raids.js:119
        console.debug("[raid] raid complete -- " + raid.team_verified.length + " of " + raid.team.length);
                ^
TypeError: console.debug is not a function at Timeout.setTimeout [as _onTimeout] 

[13:52] info: Executing command: PART #dugstanat
[13:52] info: [#trugamer_maniac] <matrix_agent>: Great raid team! :D You just made the whole team more awesome!
[13:52] info: [#trugamer_maniac] <matrix_agent>: !bonusall 0
Done. ```

**Comment by scryptonite** [comment]
@shipofgenius 😨 Oh man, you're right! I wrongly assumed that `console.debug` was widely supported. Submitted a PR to address it: #22

Thanks!
